py/farm_ng/core/event_service.py

```diff
@@ -420,6 +447,7 @@ async def _send_raw(
         # so that the client can know that the message was not sent by checking
         # the count of the URI.
         self._counts[uri.path] = count + 1
+        self._metrics.service_counts[uri.path] = count + 1
```

Comment:
Here i imagine the metric "key" to be f"{uri.path}/send_count", or similar.  Likely metrics are simple key value pairs, with a set of relatively constrained value types.

Good inspiration can be found from Prometheus: https://prometheus.io/docs/concepts/metric_types/

Imagine as a developer I want to create an eventservice metrics to prometheus bridge, to dashboard the health of the system.